From dc19d8552bc0b6e0261fe7d28be81fe808a659e8 Mon Sep 17 00:00:00 2001
From: Dave Stevenson <dave.stevenson@raspberrypi.org>
Date: Fri, 5 Apr 2019 17:24:20 +0100
Subject: [PATCH] drm: vc4: Remove now unused structure.

Cleaning up structure that was unused after
fbb59a2 drm: vc4: Add an overlay plane to vc4-firmware-kms

Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.org>
---
 drivers/gpu/drm/vc4/vc4_firmware_kms.c | 19 -------------------
 1 file changed, 19 deletions(-)

--- a/drivers/gpu/drm/vc4/vc4_firmware_kms.c
+++ b/drivers/gpu/drm/vc4/vc4_firmware_kms.c
@@ -70,25 +70,6 @@ struct mailbox_set_plane {
 	struct set_plane plane;
 };
 
-struct fb_alloc_tags {
-	struct rpi_firmware_property_tag_header tag1;
-	u32 xres, yres;
-	struct rpi_firmware_property_tag_header tag2;
-	u32 xres_virtual, yres_virtual;
-	struct rpi_firmware_property_tag_header tag3;
-	u32 bpp;
-	struct rpi_firmware_property_tag_header tag4;
-	u32 xoffset, yoffset;
-	struct rpi_firmware_property_tag_header tag5;
-	u32 base, screen_size;
-	struct rpi_firmware_property_tag_header tag6;
-	u32 pitch;
-	struct rpi_firmware_property_tag_header tag7;
-	u32 alpha_mode;
-	struct rpi_firmware_property_tag_header tag8;
-	u32 layer;
-};
-
 struct mailbox_blank_display {
 	struct rpi_firmware_property_tag_header tag1;
 	u32 display;
